sssom/util.py

```diff
@@ -824,15 +824,13 @@ def get_prefixes_used_in_table(df: pd.DataFrame):
 
 def filter_out_prefixes(df: pd.DataFrame, filter_prefixes) -> pd.DataFrame:
     rows = []
+    
     for index, row in df.iterrows():
-        ok = True
-        for col in _defining_features:
-            v = row[col]
-            prefix = get_prefix_from_curie(v)
-            if prefix in filter_prefixes:
-                ok = False
-                break
-        if ok:
+        # Get list of CURIEs from the 3 columns (_defining_features) for the row.
+        list_of_curies = [get_prefix_from_curie(row_values) for row_values in row[_defining_features]]
```

Comment:
would you please rename `_defining_features` to something more descriptive? Maybe also add a docstring to go with that constant like

```python
#: Docstring for variable
_defining_features = ...
```

Reply:
also if all of the things in the row that are being extracted are CURIEs, then it wold be more descriptive to rename `row_values` to `curie`

Reply:
further, naming variables based on their datatypes is redundant. you could rename `list_of_curies` to just `curies`

Reply:
Finally, since the goal is to check membership later, it would be better to use a set comprehension instead of a list comprehension. That can be done with the squiggly brackets instead of the square ones.

Reply:
Thanks @cthoyt for these detailed reviews! We will take care of this :)

Reply:
I concur!